client/src/main/proto/spine/event_annotations.proto

```diff
@@ -36,8 +36,36 @@ option java_package = "org.spine3.annotations";
 
 import "google/protobuf/descriptor.proto";
 
+extend google.protobuf.MessageOptions {
+    // An instruction to associate an enrichment message with the event message to enrich.
+    //
+    // For example:
+    // <pre>
+    // message MyEventEnrichment {
+    //     option (enrichment_for) = "MyEvent";
+    //     ...
+    // }
+    // </pre>
+    //
+    // If the enrichment message is enclosed into the target event message, this option can be omitted.
+    //
+    // If the event message to enrich is in another package, the type name prefixing the field reference
+    // must be fully-qualified:
+    // <pre>
+    // message MyEventEnrichment {
+    //     option (enrichment_for) = "com.company.MyEvent";
+    //     ...
+    // }
+    // </pre>
```

Comment:
Please also document the case when a reference in `by` can be qualified with a type name.